util.js

```diff
@@ -43,7 +44,7 @@ function urlTemplate (opts) {
   var hostMirrorUrl = getHostMirrorUrl(opts)
```

Comment:
There's a `v{version}` on line 43 as well

Reply:
I think it is correct not to change this line.

In my project my dependency `@serialport/bindings` fails to install.

It calls `prebuild-install --tag-prefix @serialport/bindings@`.

The correct download url is https://github.com/node-serialport/node-serialport/releases/download/@serialport/bindings@2.0.2/bindings-v2.0.2-node-v57-linux-x64.tar.gz

As you see the `v` in `v2.0.2-node...` must not be replaced by `@serialport/bindings@`.

Reply:
> As you see, `v` in `v2.0.2-node...` must not be replaced by `@serialport/bindings@`.

:+1: